packages/core/src/components/tree-view/readme.md

```diff
@@ -52,6 +52,21 @@ The consumer of the Tree View should update the TreeView's collection on the ope
 | `openItem`  | Emitted when a tree view item is opened. The `detail` property of the `CustomEvent` will contain the complete path of TreeViewItems from the root to the item that is emitting the open event. The consumer of the event is responsible for updating the TreeView's collection (usually set the open state on the last TreeViewItem in path).                                                                                                                   | `CustomEvent<TreeViewItem[]>`       |
 
 
+## TreeViewPointerEvent
```

Comment:
Op regel 36 staat `<!-- Auto Generated Below -->`, tijdens builden wordt dit opnieuw gegenereerd. Zie ook de Docs in het Storybook component: https://core.dso-toolkit.nl/_1433-Tree-View-lange-labels-vallen-onder-caret/?path=/docs/tree-view--tree-view

Reply:
Niet goed op gelet, maar hoe doen we dit dan netjes? Naar boven verplaatsen?